### PR TITLE
scala 2.13.12

### PR DESCRIPTION
**Pull request by pjfanning**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -21,6 +21,7 @@ sourceDistName := "apache-pekko-grpc"
 sourceDistIncubating := true
 ThisBuild / versionScheme := Some(VersionScheme.SemVerSpec)
 ThisBuild / resolvers += Resolver.ApacheMavenSnapshotsRepo
+ThisBuild / resolvers += Resolver.ApacheMavenStagingRepo
```
Comment (mdedetrich) [inline build.sbt:24]: How is this related to this PR?

Comment (pjfanning) [reply build.sbt:24]: needed to fix the build - the mima check is finding the v1.0.2-RC1 tag and trying to pull the 1.0.2-RC1 jars

Comment (mdedetrich) [reply build.sbt:24]: Ah I see

**Review by mdedetrich — APPROVED** [review]
lgtm
